src/cutadapt/config.vsh.yaml

```
@@ -0,0 +1,446 @@
+        - name: --max_expected_errors
+          alternatives: [--max_ee]
+          type: long
+          description: |
+            Discard reads whose expected number of errors (computed
+            from quality values) exceeds ERRORS.
+        - name: --max_average_error_rate
+          alternatives: [--max_aer]
+          type: long
+          description: |
+            as --max_expected_errors (see above), but divided by
+            length to account for reads of varying length.
+        - name: --discard_trimmed
+          alternatives: [--discard]
+          type: boolean_true
+          description: |
+            Discard reads that contain an adapter. Use also -O to
+            avoid discarding too many randomly matching reads.
+        - name: --discard_untrimmed
+          alternatives: [--trimmed_only]
+          type: boolean_true
+          description: |
+            Discard reads that do not contain an adapter.
+        - name: --discard_casava
+          type: boolean_true
+          description: |
+            Discard reads that did not pass CASAVA filtering (header
+            has :Y:).
+
+    ####################################################################
+    - name: Output parameters
+      arguments:
+        - name: --report
+          type: string
+          choices: [full, minimal]
+          description: |
+            Which type of report to print: 'full' or 'minimal'.
+          default: full
+        - name: --json
+          type: boolean_true
+          description: |
+            Write report in JSON format to report.json in
+            the output directory.
+        - name: --output
+          type: file
+          description: |
+            Write trimmed reads to this directory and name the files using {name}.
+            FASTQ or FASTA format is chosen depending on input.
+            Summary report is sent to standard output.
+          default: output
+          direction: output
+          required: true
+          must_exist: true
+        - name: --fasta
+          type: boolean_true
+          description: |
+            Output FASTA to standard output even on FASTQ input.
+        - name: --info_file
+          type: boolean_true
+          description: |
+            Write information about each read and its adapter matches
+            into info.txt in the output directory.
+            See the documentation for the file format.
+        # - name: -Z
+        # - name: --rest_file
+        # - name: --wildcard-file
+        # - name: --too_short_output
+        # - name: --too_long_output
+        # - name: --untrimmed_output
+        # - name: --untrimmed_paired_output
+        # - name: too_short_paired_output
+        # - name: too_long_paired_output
+  resources:
+    - type: bash_script
+      path: script.sh
+  test_resources:
+    - type: bash_script
+      path: test.sh
+    - type: file
+      path: test_data
+platforms:
+  - type: docker
+    image: python:3.8
```

Would python:3.12 also work?

```suggestion
    image: python:3.12